components/script/webdriver_handlers.rs

```diff
@@ -529,6 +530,40 @@ pub fn handle_get_attribute(
         .unwrap();
 }
 
+#[allow(unsafe_code)]
+pub fn handle_get_property(
+    documents: &Documents,
+    pipeline: PipelineId,
+    node_id: String,
+    name: String,
+    reply: IpcSender<Result<WebDriverJSValue, ()>>,
+) {
+    reply
+        .send(match find_node_by_unique_id(documents, pipeline, node_id) {
+            Some(node) => {
+                let cx = documents.find_document(pipeline).unwrap().window().get_cx();
+
+                rooted!(in(cx) let mut property = UndefinedValue());
+                match unsafe {
+                    get_property_jsval(
+                        cx,
+                        node.reflector().get_jsobject(),
+                        &name,
+                        property.handle_mut(),
+                    )
+                } {
+                    Ok(_) => match unsafe { jsval_to_webdriver(cx, property.handle()) } {
+                        Ok(property) => Ok(property),
+                        Err(_) => Ok(WebDriverJSValue::Null),
+                    },
+                    _ => Ok(WebDriverJSValue::Null),
```

Comment:
We should call `throw_dom_exception` on the Error value if we get an Err from `get_property_jsval`. We should probably return Undefined instead of null for the error cases, too.